riotfile.py

```diff
@@ -2176,5 +2176,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="futures",
+            command="pytest {cmdargs} tests/contrib/futures",
+            venvs=[
+                Venv(pys=["2.7"], pkgs={"futures": ["~=3.0", "~=3.1", "~=3.2", "~=3.4"]}),
```

Comment:
do we need to test all of these versions?

Reply:
if we do want to stop testing certain versions, we should follow a process akin to deprecating support.

Reply:
it's cheap to test them and there hasn't been any maintenance burden...  so 🤷‍♂️